package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bpmn-js",
-  "version": "17.9.2",
```

Comment:
Please remove this change. The version will be updated once we release. We will talk about this later. 😉